maintainers/scripts/nix-diff.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
I see other scripts here using `/usr/bin/env bash` but perhaps this should be using a nix-shell shebang?
The 'fetch-kde-qt.sh' script is a good example:

https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/fetch-kde-qt.sh#L1

It also ensures you have whatever dependencies you need, like the coreutils bit you mention.

Since this script uses nix tools already it seems safe to assume nix-shell is available :).